### ISSUE TITLE
collar locks up repeatedly upon tp

### ISSUE THREAD

**Issue by Akinyimarie** [OpenCollar/opencollar#57]
Recently with the past 2 updates, almost every time i teleport, i get stuck either in position or unable to put myself into position. The ONLY way to fix this is to re-log. I can't "reset' scripts as has been suggested because that knocks my owner off my collar and unless he's online, i can't re-add him as the collar states that no one by that name is nearby to be added. I don't know what else to do, but re-logging every time is getting really really old.

**Comment by WendyStarfall** [comment]
Does the collar tell you it lost permission to animate you?

**Comment by Akinyimarie** [comment]
No, it doesn't give any message. Simply locks up and i can't access anything nor can my Master 

----- Original Message -----

**Comment by WendyStarfall** [comment]
Okay, my best guess is that you end up teleporting to a No Script zone, the collar gets paused, you relog back home it works again, teleport and it happens all over. If you contact me in-world, I can pass you a script that will make the collar run in No Script zones similar to an AO and if you don't wanna mess with build tools, I can put it in an updater for you.

**Comment by Akinyimarie** [comment]
i'm not teleporting into a No Script Zone. The collar gets locked within the same sim which allows scripts all over. i relog into the exact same spot and place and it works. 

----- Original Message -----

**Comment by Akinyimarie** [comment]
It is still locking up and I'm noticing others saying they are having a similar issue. Sending a script to allow to work in an unscripted sim isn't going to fix the issue when that isn't the issue. Please look into this further as it is still annoying to have to constantly relog to fix the problem. Thanks

**Comment by WendyStarfall** [comment]
Hello Akinyimarie,

this issue couldn't be reproduced. I tried to make it happen when you opened the report and couldn't, so I had to guess and the only sound guess would be a no-script zone which turned out not to be the reason.

You will have to let me look at it in-world because "almost every time we teleport, we get stuck either in position or unable to put ourselves into position" is not happening with the stock software bundle. There must be a variable we don't know and the quickest and most simple way to solve this would be inspecting the collar itself.

No worries, we'll fix that for you. =)
